Sources/Networking/AlamofireNetworkClient/AlamofireNetworkClient.swift

```diff
@@ -22,7 +22,7 @@ public typealias ProgressHandler = Alamofire.Request.ProgressHandler
 public typealias ErrorHandler = (Swift.Error, Data) throws -> Swift.Error
 
 open class AlamofireNetworkClient {
-  private let session: Alamofire.Session
+  public let session: Alamofire.Session
```

Comment:
You could create a new client with a different session instead. Would that work for your use-case?

Reply:
That was the initial idea, but for our use case its not possible without changing the core logic of our Network Layer / Package. :/

Reply:
Thinking.... 🤔
We could eventually inject required state to the project package instead, and go without this as well. What do you think @dejanlatecheckout?

Reply:
This would work, let me try it out.